packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -0,0 +1,11 @@
+// @internal
+export type PluginsExtensionConfig = {
```

Comment:
Sadly these need to be placed in grafana-data due to the config interface is defined there.

Reply:
Does it need to be included in the config interface or is it enough to define it in `GrafanaBootConfig` in @grafana/runtime?

Reply:
This is a great suggestion. The config used when we are bootstrapping the app isn't relying on the interface. So I will do this change. Thanks!